modules/nf-core/shigapass/main.nf

```diff
@@ -0,0 +1,69 @@
+process SHIGAPASS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/shigapass:1.5.0--hdfd78af_0':
+        'biocontainers/shigapass:1.5.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("${prefix}.tsv"), emit: report
```

Comment:
Can you align the commands please (and I would put the optional: true after the emit:)